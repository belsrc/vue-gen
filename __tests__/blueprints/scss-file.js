const genScss = require('./../../src/blueprints/scss-file');

const className = 'test-component';

describe('genScss', () => {
  test('has the expected ouput structure', () => {
    const actual = genScss(className);

    expect(actual).toMatchSnapshot();
  });
});
