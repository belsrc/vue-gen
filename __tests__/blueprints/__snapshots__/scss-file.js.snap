// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`genScss has the expected ouput structure 1`] = `
"// Mobile / Baseline
.test-component {}

// Tablets
@media only screen and (min-width: 768px) {
  .test-component {}
}

// Hi-Res Tablets
@media only screen and (min-width: 1024px) {
  .test-component {}
}

// Desktops
@media only screen and (min-width: 1200px) {
  .test-component {}
}

// Mid DPI Laptop/Desktop
@media only screen and (min-width: 1360px) {
  .test-component {}
}

// Hi DPI Desktop
@media only screen and (min-width: 1440px) {
  .test-component {}
}

// Larger Desktops
@media only screen and (min-width: 1920px) {
  .test-component {}
}
"
`;
